modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -294,7 +295,11 @@ const Main = ({
 
 	const isSignedIn = Liferay.ThemeDisplay.isSignedIn();
 
-	const getErrorMessages = (errorMessage, isSignedIn) => {
+	const getErrorMessages = (
+		errorMessage,
+		objectInvalidExtension,
+		isSignedIn
+	) => {
```

Comment:
Hi @nicolas-sss, Could you please sort alphabetically and rename `objectInvalidExtension` to `objectFieldInvalidExtension`.